tests/tests.py

```diff
@@ -108,6 +108,10 @@ def test15():
     run(dpath("test15"))
 
 
+def test_glpk_solver():
+    run(dpath("test_glpk_solver"), scheduler_ilp_solver="GLPK_CMD")
```

Comment:
Please note that `GLPK_CMD` is the default anyway whenever `glpk` is installed (and no bundled Cbc/GLPK is available, which should always be the case for the conda-forge package).
Meaning, by adding `glpk` to the `test-environment.yml`, the tests will run with GLPK but not with Cbc, albeit the latter is probably the one that most (non-Windows) users will run, currently.

Reply:
Good point. I just checked and the only solver available when `glpk` is not installed is `COIN_CMD`.

Reply:
It appears as `COIN_CMD` is not available by default on windows. Due to the lack of a windows environment I can't check what's the default solver.

Reply:
But you should add tests for both, e.g.,
```python
def test_glpk_solver():
    run(dpath("test_coin_solver"), scheduler_ilp_solver="GLPK_CMD")

# But run the one below only for Unix, because Cbc is currently not packaged on Windows.
def test_coin_solver():
    run(dpath("test_coin_solver"), scheduler_ilp_solver="COIN_CMD")
```
Or (maybe even better?) run the tests for all solvers from `ilp_solvers = pulp.list_solvers(onlyAvailable=True)`, but then make also sure that `assert "GLPK_CMD" in ilp_solvers` and also (on Unix) `assert "COIN_CMD" in ilp_solvers`.